### ISSUE TITLE
Pi-hole logs clients when in anonymous mode

### ISSUE THREAD

**Issue by CorruptComputer** [pi-hole/pi-hole#3432]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

0

---
**Expected behaviour:**

Pi-hole should not be logging network clients when in anonymous privacy mode. 

**Actual behaviour:**

Under the query logs I can see that the IP's are replaced with 0.0.0.0, however going to the Network tab I can still see the IP addresses which are connecting to my Pi-Hole server.

**Steps to reproduce:**

1. Turn on anonymous mode.
2. Send some DNS queries to the server.
3. Check network tab, and see the IP's they are coming from.

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**  

https://tricorder.pi-hole.net/wdd966jdt1

**Troubleshooting undertaken, and/or other relevant information:**

Tried flushing logs, flushing network table, and restarting resolver service. After flushing those the list will be empty for a few seconds and start filling up again.

**Comment by dschaper** [comment]
Hello, you are running an open resolver and we provide no support for that.

**Comment by CorruptComputer** [comment]
Because I run this resolver in the cloud instead of on my network you are saying that I can't open an issue for a problem in it. Are you fucking kidding me? That is not in the contributors guidelines anywhere.

**Comment by dschaper** [comment]
No, I'm not fucking kidding you.

**Comment by CorruptComputer** [comment]
Why is it that I can't report an issue with Pi-Hole if I run it in the cloud? I want to be able to access it on my phone when on mobile connection.

Why is that so bad that you won't even accept an issue request for something that isn't working properly with your software? Especially when it is not in the contributing guidelines, not in the readme, and not in any sort of rules list. I just genuinely don't understand it.

**Comment by dschaper** [comment]
You are running an open resolver, you are publicizing that it's an open resolver. If you don't understand why you shouldn't be running an open resolver then you don't need to be using Pi-hole as an open resolver.

**Comment by dschaper** [comment]
And the issue you are reporting is "works as intended." The network interface page is a web representation of `ip neigh` and the `arp` cache of the Pi-hole interface.

**Comment by dschaper** [comment]
What is this for then?
![Screenshot_2020-05-23 Nickolas Gupton](https://user-images.githubusercontent.com/2782850/82739622-c1c42800-9cf5-11ea-9b68-4669550fa762.png)

**Comment by PromoFaux** [comment]
[Why you shouldn't run an open resolver](https://www.cloudflare.com/learning/ddos/dns-amplification-ddos-attack/)

If you want to be able to access your pi-hole whilst on the go, you should be using a VPN, else you run the risk of being responsible for DDOS attacks. We cannot and will not encourage this.

https://pi-hole.net/2016/09/15/tips-for-accessing-your-pi-hole-remotely/#respond

https://docs.pi-hole.net/guides/vpn/overview/

**Comment by CorruptComputer** [comment]
@dschaper I'm not sure I understand your question, it clearly says what its for. If you're asking about the screenshot I changed the privacy level to be higher today and saw that it still logged IP's. Which is why I opened the issue. Yeah I get that it might not just be me using it, and I can't block http/https on the server unless I don't need the admin interface (which I do). Which is why I placed that web page into the lighttpd directory, in case any of my friends and family asked me if they could use it too.
